daq/host.py

```diff
@@ -437,7 +439,9 @@ def terminate(self, reason, trigger=True):
         self._monitor_cleanup()
         if self.target_port:
             self.runner.network.delete_mirror_interface(self.target_port)
-        self._finalize_report()
+
+        test_results = self._finalize_report()
```

Comment:
I think if you were to add an attribute “test_results” to the Device class and directly store the results on self.device, it would’ve made the runner class cleaner.

Reply:
So I see we access the host in runner.py through `runner.device.host`. And in host.py, we access the device thorugh  `host.device`. Are the two `device` the same object?